Cogs.Publishers/JsonSimpleConverter.cs

```diff
@@ -31,7 +31,7 @@ public override void WriteJson(JsonWriter writer, object value, JsonSerializer s
                 obj.Add(new JProperty("dateTime", 
                     new JObject(
                         new JProperty("type", "string"), 
-                        new JProperty("format", "date-time"))));
+                        new JProperty("pattern", @"^[0-9][0-9][0-9][0-9]-(0?[1-9]|1[0-2])-(0?[1-9]|[12][0-9]|3[01])([A-Z])(00|0[0-9]|1[0-9]|2[0-3]):([0-9]|[0-5][0-9]):([0-9]|[0-5][0-9])(\+)(00|0[0-9]|1[0-9]|2[0-3]):([0-9]|[0-5][0-9]):([0-9]|[0-5][0-9])$"))));
```

Comment:
We can use the string format "yyyy-MM-dd\THH:mm:ss.FFFFFFFK" in c# to read in and write out the iso 8601 time format